tests/virt_autotest/xen_guest_irqbalance.pm

```diff
@@ -28,7 +29,7 @@ sub run_test {
     foreach my $guest (keys %virt_autotest::common::guests) {
 
         #irqbalance test run on sle12sp3+ guest
-        next if guest_is_sle($guest, '<=12-sp2');
+        next if (guest_is_sle($guest, '<=12-sp2') || guest_is_sle($guest, '=15-sp0'));
```

Comment:
Can you add a record_soft_fail or record_info for 15sp0 with bug id and reason to skip?

Reply:
Ok, alreay added. please see the new push: http://openqa.qam.suse.cz/tests/38168#step/xen_guest_irqbalance/2584